test/combined_dataset_test.py

```diff
@@ -63,7 +63,7 @@ def test_combined_county_has_some_timeseries_data(fips):
     region = Region.from_fips(fips)
     latest = combined_datasets.load_us_timeseries_dataset().get_one_region(region)
     df = latest.data.set_index(CommonFields.DATE)
-    date = "2020-09-01"  # Arbitrary date that both FIPS have data for.
+    date = "2020-09-04"  # Arbitrary date that both FIPS have data for.
```

Comment:
This was to fix this test for the HHS county data (had to pick the day within the week that they collected data). Since we're punting on county data it's not needed, but it's non-harmful and will save re-making the change later.